src/rest/connector/libs/nxos/implementation.py

```diff
@@ -418,5 +418,5 @@ def put(self, dn, payload, headers=None, timeout=30, **kwargs):
         Raises:
             RequestException if response is not ok
         """
-        return self._request('DELETE', dn, data=payload, headers=headers,
```

Comment:
@danielgraziano this looks like forgotten to update. but let me know if `DELETE` was intended with reason.